internal/server/node/golden/get_property_labels_test.go

```diff
@@ -68,25 +68,30 @@ func TestGetPropertyLabels(t *testing.T) {
 				t.Errorf("could not GetPropertyLabels: %s", err)
 				continue
 			}
-			var result map[string]*model.PropLabelCache
-			err = json.Unmarshal([]byte(resp.GetPayload()), &result)
+			// Here the golden file is not same as the actual API output.
+			// An addtional level of "data" is added to make the proto<->json conversion
+			// doable.
+			payload := resp.GetPayload()
+			payload = "{\"data\":" + payload + "}"
```

Comment:
nit: line 74-75:

payload := "{\"data\":" + resp.GetPayload() + "}"

Reply:
ditto for other test files.

Reply:
Done